2021q3/cpe.adoc

```diff
@@ -0,0 +1,49 @@
+== Improve CPE Data in the Portstree
```

Comment:
"ports tree", I think. (And same for the other occurences below.)

Reply:
I had a look around and found that we use the term "FreeBSD Ports Collection" [1] so I decided to use that.

[1] https://www.freebsd.org/ports/